app/build.gradle

```diff
@@ -38,35 +38,35 @@ android {
 }
 
 dependencies {
-    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    //kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$Versions.Tools.kotlin"
-    implementation 'androidx.appcompat:appcompat:1.1.0-alpha01'
-    implementation 'androidx.recyclerview:recyclerview:1.1.0-alpha01'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-alpha3'
-    implementation 'androidx.paging:paging-runtime:2.1.0-rc01'
+
+    //android
     implementation "android.arch.paging:rxjava2:$Versions.pagingRx"
-    implementation "io.reactivex.rxjava2:rxjava:$Versions.rxJava"
-    implementation "io.reactivex.rxjava2:rxandroid:$Versions.rxAndroid"
-    implementation "com.jakewharton.rxbinding3:rxbinding:$Versions.rxBinding"
+    implementation "androidx.appcompat:appcompat:$Versions.androidX"
+    implementation "androidx.constraintlayout:constraintlayout:$Versions.constraintLayout"
+    implementation "androidx.lifecycle:lifecycle-extensions:$Versions.lifecycleEx"
+    implementation "androidx.paging:paging-runtime:$Versions.paging"
+    implementation "androidx.recyclerview:recyclerview:$Versions.androidX"
     implementation 'com.google.android.material:material:1.1.0-alpha03'
-//    implementation "com.squareup.retrofit2:retrofit:$Versions.retrofit"
-//    implementation "com.squareup.retrofit2:adapter-rxjava2:$Versions.retrofit"
-//    implementation "com.squareup.retrofit2:converter-gson:$Versions.retrofit"
+
+    //deps
     implementation "com.github.bumptech.glide:glide:$Versions.glide"
     kapt "com.github.bumptech.glide:compiler:$Versions.glide"
-    implementation "com.facebook.stetho:stetho:$Versions.stetho"
```

Comment:
I want to use stetho, can we keep this as `debugImplementation`?

Reply:
we are gonna use the one from Fuel?